ssl/tls13_enc.c

```diff
@@ -516,10 +516,12 @@ int tls13_change_cipher_state(SSL *s, int which)
     const EVP_MD *md = NULL;
     const EVP_CIPHER *cipher = NULL;
 #if !defined(OPENSSL_NO_KTLS) && defined(OPENSSL_KTLS_TLS13)
-# ifndef __FreeBSD__
+# ifdef __FreeBSD__
+    struct tls_enable crypto_info;
+# else
     struct tls_crypto_info_all crypto_info;
```

Comment:
We seem to have done this in several places already - and here is another instance. Perhaps it would be better to introduce our own typedef for this - the definition of which can have the `__FreeBSD__` guards around it - and then from then on we just use that typedef without having to special case FreeBSD everywhere.